README.md

```diff
@@ -68,29 +68,67 @@ brew install k6
 
 ### Windows
 
-You can manually download and install the [official `.msi` installation package](https://dl.bintray.com/loadimpact/windows/k6-v0.31.1-amd64.msi) or, if you use the [chocolatey package manager](https://chocolatey.org/), follow [these instructions](https://bintray.com/repo/buildSettings?repoPath=%2Floadimpact%2Fchoco) to set up the k6 repository.
+If you use the [Chocolatey package manager](https://chocolatey.org/):
```

Comment:
Since we haven't taken over this package yet, we should add a disclaimer here that it's not an official package.

Reply:
Sure, see 7e74e888. Not a real disclaimer, just a mention of "unofficial", which I think is good enough since we emphasize "official" for the MSI.

Though I think either way should be fine since we'll be taking it over in a few weeks anyway.